pulseapi/creators/models.py

```diff
@@ -0,0 +1,32 @@
+"""
+The creator field for an entry. Can be empty, just a name, or linked to a pulse user
+"""
+from django.db import models
+from pulseapi.users.models import EmailUser
+
+class CreatorQuerySet(models.query.QuerySet):
+    """
+    A queryset for creators which returns all creators by name
+    """
+
+    def public(self):
+        """
+        Returns all creators to start
```

Comment:
what does this mean? this function definitely doesn't do what this comment says, it's just a reflection function for get a reference to an object (to which you would already have a reference, so not sure what this function is for. What uses this?)